streams/src/test/java/org/apache/kafka/streams/processor/internals/StreamTaskTest.java

```diff
@@ -365,11 +356,10 @@ public void shouldResetOffsetsToLastCommittedForSpecifiedPartitions() {
         consumer.seek(partition1, 10L);
         consumer.seek(partition2, 15L);
 
+        @SuppressWarnings("unchecked")
         final java.util.function.Consumer<Set<TopicPartition>> resetter =
-            EasyMock.mock(java.util.function.Consumer.class);
-        resetter.accept(Collections.emptySet());
-        EasyMock.expectLastCall();
-        EasyMock.replay(resetter);
+            mock(java.util.function.Consumer.class);
+        doNothing().when(resetter).accept(Collections.emptySet());
```

Comment:
Again, I do not think we need this stub with Mockito.

Reply:
This is fair, I have added a verification instead since it was expected in EasyMock. If you don't think the verification is necessary I can remove it completely